docs/source/tutorials/plots_intro.ipynb

```diff
@@ -530,6 +530,7 @@
    "id": "5211f6de-d447-46a8-8652-3c9fd77d7915",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        aes_map={"point_estimate": ["color", "overlay", "alpha"]},
</pre><p>I'm struggling to understand this. But I confess I have not read this very thoroughly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz-plots/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>